build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'application'
+    id 'jacoco'
+}
+
+group = 'com.qumu.test'
+version = '0.0.1-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+java {
+    sourceCompatibility = JavaVersion.toVersion("11")
```

Comment:
I'd be tempted to use a newer version of Java. I feel we should be moving forwards here in general for our products - containerisation and certainly means we shouldn't restrict ourselves!

But version 11 if fine for this too.